crates/ruma-federation-api/src/discovery/get_remote_server_keys_batch/v2.rs

```diff
@@ -20,14 +20,6 @@ ruma_api! {
     }
 
     request: {
-        /// The time until which the returned certificates will need to be valid to be useful to
-        /// the requesting server.
-        ///
-        /// If not supplied, the current time as determined by the notary server is used.
-        #[ruma_api(query)]
-        #[serde(default = "MilliSecondsSinceUnixEpoch::now")]
-        pub minimum_valid_until_ts: MilliSecondsSinceUnixEpoch,
```

Comment:
Ugh, the spec is so hard to read sometimes...

Is the field just not required but is available? If so could we make it `Option<..>`.

Reply:
It already has a `serde(default)`. But it looks like this field is *only* meant to exist on the `QueryCriteria` struct below, which.. uses `Option`. @gnieto can you transfer the `serde(default)` from here on that to replace the `Option` in a separate commit?

Reply:
Ahh right it's the `Request` we are looking at :facepalm:

Reply:
@DevinR528 As @jplatte said, this field does not exists on this request, so I think it's better to just remove it.

@jplatte I can do that, but this implies that if this endpoint is used from a client perspective, you won't be able to let the remote server select the current timestamp as the `minimum_valid_until_ts`, isn't it?

What I mean is that, if I change this from `Option<MilliSecondsSinceUnixEpoch>` to `MilliSecondsSinceUnixEpoch`, I will need to (manually) implement `Default` for `QueryCriteria`, assigning `minimum_valid_until_ts` to `MilliSecondsSinceUnixEpoch::now()`, which will set the client timestamp instead of the server one. I'm not sure that makes any difference, but I wanted to share it just in case.

I already did that change on a local branch, but I have not pushed it because I was not sure if this behaviour was desired. Let me know if you want me to push it.

Reply:
Oh, the default being determined by client instead of server is a very good point here, with the `serde(default)` not being deterministic. Let's keep things as they are.